shapely/geoarrow.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+from shapely._geoarrow import ArrayReader, GeoArrowGEOSException  # NOQA
+
+
+def from_arrow(arrays, schema):
```

Comment:
Can we let `arrays` be either a single array or multiple? In the single case we'd check for `__arrow_c_array__` on the input directly?

In the future, this could check for `__arrow_c_stream__` for a chunked array? 

`schema` could also be optional here, in which case you just assert that the schemas of each array chunk are the same? Or, really, there's no reason to disallow arrays with differing schemas, right? So it seems preferable to load the schema from `__arrow_c_array__` anyways, unless instantiating `ArrayReader` is expensive.

Reply:
Presumably this needs a non-table stream, right? I.e. it'll error on struct input that doesn't conform to points?